### ISSUE TITLE
Some details about the pre-trained model

### ISSUE THREAD

**Issue by liu-zhenhua** [pudae/tensorflow-densenet#19]
Hi, thanks for your code.
I have two confusions about the pre-trained model. 
1. The order of input channels is RGB or BGR?
2. I see the dropout_rate is set to None in the code and did you use dropout layer in the pre-trained model?
Thanks again and look forward to your apply!
